site/content/blog/2022-07-01-whats-new-in-svelte-july-2022.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "What's new in Svelte: July 2022"
+description: "Faster SSR, language tools improvements and something special..."
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With [something special](https://twitter.com/SvelteSociety/status/1539284972544180230?s=20&t=3Qoi371NrL9LP6xMOF63ag) dropping today along with faster SSR and support for Vitest and Storybook in SvelteKit, there's a lot to cover in this month's newsletter...
```

Comment:
I think I'd wait until it's actually dropped before releasing the new blog post 😄 otherwise great as always!

Reply:
I've got some copy in the wings waiting to plug in here ;) Will drop it in right before the merge on the first :)

Reply:
After a conversation with @kevmodrome, we're going to take out the "something special": https://github.com/sveltejs/svelte/pull/7637/commits/ad6114d28e3c0bb9cbecf4c920098cb99d1b0958

He needs more time to polish it up ;)